### PR TITLE
[ft #162572369]Add view all and one article

### PR DESCRIPTION
**Pull request by bozicschucky**
#### What does this PR do?
Enables a user to be able to view many or a particular article.

#### Description of Task to be completed?
Users should be able to view one or many articles.

#### How should this be manually tested?
 - Clone the repository.
 - Install application dependencies using`yarn`
 - start the server using `yarn start:dev`
 - Enter `localhost:8080/` in the browser
 - A user should be able to view all the articles created by other authors and on click on one of them, they should be able to view the details of that particular article.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#162572369

### CODE REVIEW

**Review by johnkegz — APPROVED** [review]
Looks good

File: tests/integerationTests/ArticlesView.test.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          />
+        </Router>
+      </Provider>
+    );
+    const instance = wrapper.instance();
+    expect(wrapper.find(".card-body").find("#component-btn").first().simulate('click'))
+  });
+});
+
+// describe("<ArticleDetails with shallow/>", () => {
```
Comment (shakirandagire) [inline tests/integerationTests/ArticlesView.test.jsx]: Could you remove the commented out code from this file

File: src/components/Header.jsx [hunk 2]
```diff
@@ -7,9 +7,11 @@ const Header = () => (
   <header>
     <nav className="navbar fixed-top navbar-expand-lg navbar-dark unique-color-dark scrolling-navbar">
       <div className="container">
-        <a className="navbar-brand" href="/index.html">
+        {/* <a className="navbar-brand" href="index.html"> */}
```
Comment (shakirandagire) [inline src/components/Header.jsx]: Could you remove the commented out code

File: tests/integerationTests/ArticlesView.test.jsx (hunk 1 above)
Comment (bozicschucky) [reply tests/integerationTests/ArticlesView.test.jsx]: I have removed the commented out code.

File: src/components/Header.jsx (hunk 2 above)
Comment (bozicschucky) [reply src/components/Header.jsx]: I have removed the commented out code.

**Review by cdvx — APPROVED** [review]
Worked on feedback, and removed commented out code, looks okay👍🏾

**Comment by emukungu** [comment]
Looks good

**Pull request merged**